scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -436,7 +467,43 @@ task SplitIntervals {
   }
 }
 
+task MergeBamOuts {
+  String gatk4_jar
+  Array[File] bam_outs
+  File picard_jar
+  File ref_fasta
+  File ref_fasta_index
+  File ref_dict
+  File? gatk4_jar_override
+  String output_vcf_name
+
+  # Runtime parameters
+  Int? mem
+  String m2_docker
+  Int? preemptible_attempts
+  Int? disk_space_gb
+
+  command <<<
+          # This command block relies that there is at least one file in bam_outs.
```

Comment:
relies --> assumes

Reply:
And you can enforce that assumption with `Array[File]+`

Reply:
Done and done